web/source/osk/bannerManager.ts

```diff
@@ -18,20 +20,47 @@ namespace com.keyman.osk {
 
     public static readonly DEFAULT_OPTIONS: BannerOptions = {
       persistentBanner: false,
-      enablePredictions: true
+      enablePredictions: true,
+      imagePath: ""
     }
 
     constructor() {
       // Step 1 - establish the container element.  Must come before this.setOptions.
       this.constructContainer();
 
+      // Register a listener for model change events so that we can hot-swap the banner as needed.
+      let keyman = com.keyman.singleton;
+      let device = keyman.util.device;
+
+      // TODO: Remove for production
+      let initialOptions: BannerOptions;
```

Comment:
While... okay... for testing, I'd much prefer this be eliminated now.

Just add instructions on how to set everything needed via console and we should be fine.

I mean, without adding this, it's pretty much a simple entry of `keyman.osk.banner.setBanner('suggestion')` via console.  This will immediately force the suggestion banner to be active until the keyboard is next swapped.

If the concern is testing this while integrated into the Android or iOS app, then I'd rather it be hardcoded into the app's keyboard.html pages for now, since that will begin the integration process - the apps should be managing banner display anyway.